### ISSUE TITLE
ui5-wizard: New Step Title Not Announced by Screen Reader when going to next step

### ISSUE THREAD

**Issue by GongRichard** [SAP/ui5-webcomponents#3651]
### **Bug Description**
New step title will not be announced by the screen reader when going to next step.

### **Expected Behavior**
New step title should be announced by the screen reader when going to next step.

### **Steps to Reproduce**
1. Go to https://github.wdf.sap.corp/pages/xweb/goalmanagementx/app.html?mock&noFirstTimeGuidance
2. Open JAWS or NVDA
2. Click 'Cascade Goal' button in the top-right header
3. Select one goal and click next button
4. The second step title information "‎Select Recipients" won't be announced and it will only announce the first focus element
5. Go to UI5 Wizard sample page https://sapui5.hana.ondemand.com/#/entity/sap.m.Wizard/sample/sap.m.sample.Wizard, and it will read out step title information

### **Context**
- UI5 Web Components version: 0.31.12
- OS/Platform: Windows
- Browser: Chrome
- Affected component: ui5-wizard

### **Log Output / Stack Trace / Screenshots**

![WC-Wizard](https://user-images.githubusercontent.com/13098801/129298581-ed9addfc-c834-4d38-aeef-c2d56eb42f7e.jpg)

![Ui5-Wizard](https://user-images.githubusercontent.com/13098801/129298557-9901141c-4590-4748-ba00-b9b0f5e0a566.png)

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SAP SuccessFactors

**Comment by GerganaKremenska** [comment]
Hello @SAP/ui5-webcomponents-topic-rl, 

 I am forwarding this issue to you, because the issue is reproducable.

Best Regards,
Gergana

**Comment by elenastoyanovaa** [comment]
Hello team, 

Please update the status wiki of the control, when the issue is fixed.

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-rl, @nikolay-kolarov  this is one of the ACC issues by SF
we need to complete by January. Raising the priority to High.

**Comment by niyap** [comment]
Hello Colleagues,

The fix is downported to the 0.31 branch. It will be available with the next patch released next week.

Kind Regards,
Niya

**Comment by ilhan007** [comment]
Hello @GongRichard  the fix is released with 0.31.27.

**Comment by GongRichard** [comment]
Hi @ilhan007 is the fix also be released to 1.0.x version?

**Comment by fifoosid** [comment]
Hi @GongRichard 

This issue is related to the 31 branch.
